zerver/views/streams.py

```diff
@@ -325,6 +328,19 @@ def add_subscriptions_backend(
     (subscribed, already_subscribed) = bulk_add_subscriptions(streams, subscribers,
                                                               acting_user=user_profile)
 
+    # Make acting user a stream admin, if acting user subscribed to newly created streams.
+    if user_profile.email in principals:
+        for stream in created_streams:
+            recipient = get_stream_recipient(stream.id)
+            try:
+                sub = Subscription.objects.get(user_profile=user_profile,
+                                               recipient=recipient)
+            except Subscription.DoesNotExist:
+                # If user is present in principals, then user should be subscribed to stream. But
+                # in case if not then just pass.
+                pass  # nocoverage
+            do_change_subscription_property(user_profile, sub, stream, "is_stream_admin", True)
+
```

Comment:
This loop will never thrown an exception ideally, cause if user is present in principals, then we had surely subscribed them to stream. If on some error we can't(if stream id can't found or user not defined) then we should have throw an error.
This is the reason tests are failing, cause this lines are not getting coverage. Is there any way I can test this exception? Or should I just add them #nocoverage?